Add complete example module for a (simple) configuration page for a module (#158)
PrestaShop/example-modules: demosymfonyformsimple/README.md

# Demonstration of how to use Symfony form types for Module configuration pages

## About

> This module demonstrates how to use Symfony form types for Module configuration pages.
> 
> [It is the module created by this complete guide on devdocs](https://devdocs.prestashop-project.org/8/modules/creation/adding-configuration-page-modern/).
> 
> It provides a simple configuration page for a module, with a Text Field. 

Reviewer: ```suggestion
In this module, you will learn how to utilize Symfony form types to create configuration pages for your module.

[This module has been created by following a guide from the developer documentation](https://devdocs.prestashop-project.org/8/modules/creation/adding-configuration-page-modern/).

It provides a simple configuration page for a module with a Text Field. This text field value is stored [using the Configuration component](https://devdocs.prestashop-project.org/8/development/components/configuration/). 
```

## Supported PrestaShop versions

> This module is compatible with PS 8.0.x versions only.

Reviewer: ```suggestion
This module has been tested with PrestaShop 8.
```

I think it should work with 8.1 as well. You can test.

## Requirements

Composer

## How to install

> Download or clone module into modules directory of your PrestaShop installation

Reviewer: ```suggestion
Download or clone the module into the modules directory of your PrestaShop installation.
```

> Rename the directory to make sure that module directory is named demosymfonyformsimple

Reviewer: ```suggestion
Rename the directory to make sure that the module directory is named `demosymfonyformsimple`.
```

`cd` into module's directory and run following commands:

`composer install` - to download dependencies into vendor folder

> Install module from Back Office or via CLI.

Reviewer: ```suggestion
Install the module from Back Office or via CLI.
```
